src/Raven.Pal/src/rvngetpalver.c

```diff
@@ -1,3 +1,3 @@
 #include <sys/types.h>
 #include "rvn.h"
-EXPORT int32_t rvn_get_pal_ver() { return  62022 ; }
+EXPORT int32_t rvn_get_pal_ver() { return  70812 ; }
```

Comment:
Shouldnt this be 71xxx?

Reply:
eventually, yes. Still making changes, and need to make sure we aren't consuming `7.1` in local nuget caches